### PR TITLE
Dockerfile for building LO from source

### PR DESCRIPTION
**Pull request by hakonhagland**
Added a `Dockerfile` for building LibreOffice from source. 
Also added a unit test file to the Python project. This file is currently only for demonstration, but it could help make it easier to add more unit tests in the future.

### CODE REVIEW

**Comment by blattms** [comment]
Cool. I hope @lisajulia  can take a look at the python code.

File: docker/README.md [hunk 1]
```diff
@@ -25,8 +25,11 @@ Linux host, see sections below for details on Windows and macOS.
 
 ## Scripts
 
-`Dockerfile`'s are located in the directory `docker_files`. Currently, there is only
-one Dockerfile directory, namely `ubuntu2204-lo7.5.9`.
+`Dockerfile`'s are located in the directory `docker_files`. Currently, there are two
+Dockerfile directories, namely `ubuntu2204-lo7.5.9` and `ubuntu2310-dev`.
+The first one can be used to build a Docker image with Ubuntu22.04 and LibreOffice version
+7.5.9. The second, allow you to build LibreOffice from source from a specified git tag
```
Comment (lisajulia) [inline docker/README.md:31]: allow**s**

**Review by lisajulia — APPROVED** [review]
Looks good to me :) (apart from the small typo)

File: docker/README.md (hunk 1 above)
Comment (hakonhagland) [reply docker/README.md]: @lisajulia Thanks! I have updated the PR and fixed the typo.

**Pull request merged**